main_routers/workshop_router.py

```
@@ -571,6 +571,15 @@ async def upload_preview_image(request: Request):
             "message": "文件上传失败"
         }, status_code=500)
 
+@router.get('/status')
+async def get_steam_status():
+    """检查Steamworks是否已初始化，用于前端页面加载时判断Steam状态"""
```

Ruff (0.15.7)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)